### PR TITLE
Adding API endpoints

### PR DESCRIPTION
**Pull request by gergelyke**
* Email launch
* Email copy
* Contactlist create
* Contactlist list
* Segment list

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "NODE_ENV=test node_modules/mocha/bin/mocha --reporter spec --harmony --require co-mocha $(find . -name \"*.spec.js\" -not -path \"./node_modules/*\")"
+    "test": "NODE_ENV=test mocha --harmony --require co-mocha $(find . -name \"*.spec.js\" -not -path \"./node_modules/*\")"
```
Comment (drawain) [inline package.json]: Why did you remove the relative address to the mocha? It's a `devDependency` so you can run it from `node_modules`. It's easier to run the tests in this way on environments without a globally installed mocha. We also use it in Codeship to  skip installing mocha on every deploy - Codeship caches the content of the `node_modules` in opposite to the global packages.

Comment (gergelyke) [inline package.json]: yeah, but `npm start` looks for `node_modules/.bin` and put the content to your path, so you don't need to install it globally, nor the relative address

Comment (drawain) [inline package.json]: Thx, we didn't know it.
